README.md

```diff
@@ -125,7 +140,8 @@ Write code to do the following:
 
 Write your code here:
 ```ruby
-# code here
+no clue what this is asking to do.
+
 ```
```

Comment:
The question is asking you to use ActiveRecord methods to achieve everything in the list. So the first question, which asks you to create a lamp and genie, would look something like this...

```rb
lamp = Lamp.create( wishes_remaining: 3 )
genie = Genie.create( name: "Genie" )
```

Is that enough to get you started?